eval.py

```diff
@@ -80,14 +80,30 @@ def create_dataset(embeddings):
 
 
 def main(
-    eval_embeddings_dir="embeddings/en",  # embeddings dir point to the same parquet file for testing and online eval
-    train_embeddings_dir="embeddings/en",
-    allowed_training_set="allowed_training_set.yaml",
-    eval_file="eval.yaml",
-    train_file="workdir/train.json",
-    config_file="dataperf_speech_config.yaml",
+    language="en",
+    eval_embeddings_dir=None,  # embeddings dir point to the same parquet file for testing and online eval
+    train_embeddings_dir=None,
+    allowed_training_set=None,
+    eval_file=None,
+    train_file=None,
+    config_file="workspace/dataperf_speech_config.yaml",
 ):
 
```

Comment:
if you decide to change main.py we should copy any warnings/asserts over here too (see my comments in that file for context) but not critical